src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/AlignedContig.java

```diff
@@ -14,22 +14,25 @@
 
 /**
  * Locally assembled contig:
- * its name
- * its sequence as produced by the assembler (no reverse complement like in the SAM record if it maps to '-' strand), and
- * its stripped-down alignment information.
+ *   its name
```

Comment:
When formatted as javadoc this will all appear as a single paragraph of text. To write a list in javadoc you need to use some basic HTML:
```
/**
  * Locally assembled contig:
  * <ul><li>its name
  * <li>its sequence ...
  * <li>its stripped-down ...
  * </ul>
  */
```
Note that you don't need need `</li>` as it's automatically closed by `<li>`.